src/docsfy/models.py

```
@@ -115,9 +121,20 @@ class NavGroup(BaseModel):
 class DocPlan(BaseModel):
     project_name: str
     tagline: str = ""
+    repo_type: RepoType = "app"
     navigation: list[NavGroup] = Field(default_factory=list)
     version: str | None = None
 
+    @field_validator("repo_type", mode="before")
+    @classmethod
+    def normalize_repo_type(cls, v: Any) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `v`

(ANN401)